src/screens/Loading/index.js

```diff
@@ -1,31 +1,31 @@
 import * as React from 'react';
 import {ActivityIndicator, Modal, StyleSheet, View} from 'react-native';
-import { withInteractionsManaged } from '../../components/WithInteractionManaged';
+import {withInteractionsManaged} from '../../components/WithInteractionManaged';
 import {COLORS} from '../../utils/theme';
 
 const Loading = ({visible}) => {
   return (
     <Modal transparent visible={visible}>
       <View style={styles.container}>
         <View style={styles.content}>
-          <ActivityIndicator size="large" color={COLORS.holyTosca} />
+          <ActivityIndicator size="large" color={COLORS.holytosca} />
         </View>
       </View>
     </Modal>
   );
 };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `Loading` component is a functional component that uses the `visible` prop to control its visibility. It's good practice to define default prop values for all props that are not marked as required. This can prevent potential bugs and crashes if the component is used without providing the `visible` prop.
```diff
- const Loading = ({visible}) => {
+ const Loading = ({visible = false}) => {
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->